src/pages/library/dataBrowser-utils.js

```diff
@@ -38,10 +67,14 @@ const normalizeSnapshot = snapshot => {
     _.uniqBy(_.toLower)
   )(snapshot['prov:wasGeneratedBy'])
 
-  const dataReleasePolicy = _.flow(
-    _.replace('TerraCore:', ''),
-    _.startCase
-  )(snapshot['TerraDCAT_ap:hasDataUsePermission'])
+  const dataReleasePolicy = _.getOr(
+    {
+      ...snapshotReleasePolicies.releasepolicy_other,
```

Comment:
I like the current solution of `...snapshotReleasePolicies` so we dont have to make sure these tags and labels are the same if they ever change, keep it to the same variable instead of hardcoding it twice. It's used in the side filters and it's used in the display on the details page (only reason why the description really even comes into play)